src/officehours_api/migrations/0022_auto_20210423_1948.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 3.0.14 on 2021-04-23 23:48
+
+import django.contrib.postgres.fields
+from django.db import migrations, models
+import officehours_api.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('officehours_api', '0021_auto_20201105_1353'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='queue',
+            name='phys_location',
+            field=models.CharField(blank=True, max_length=100),
+        ),
+        migrations.AlterField(
+            model_name='meeting',
+            name='backend_type',
+            field=models.CharField(choices=[['inperson', 'In Person'], ['zoom', 'Zoom'], ['bluejeans', 'BlueJeans']], default=officehours_api.models.get_default_backend, max_length=20),
+        ),
+        migrations.AlterField(
+            model_name='queue',
+            name='allowed_backends',
+            field=django.contrib.postgres.fields.ArrayField(base_field=models.CharField(choices=[['inperson', 'In Person'], ['zoom', 'Zoom'], ['bluejeans', 'BlueJeans']], max_length=20), default=officehours_api.models.get_default_allowed_backends, size=None),
+        ),
+    ]
```

Comment:
Were the migrations generated with: 
```
docker-compose run web python manage.py makemigrations --settings=officehours.makemigrations_settings
```
?
I see that the last migration to be applied had the same change, so evidently something is causing Django to see these as changes. At any rate, I don't think any change would actually happen if you applied them, and in the past, I don't think I ever made a manual change to any migration file.

Reply:
Yeah, so that's the question. Why are these extra migration steps being applied if it's not changing anything? I thought I'd ask since you're more familiar with how the backend types integrate with `models.py`.